RELEASE-NOTES.txt

```diff
@@ -19,6 +19,7 @@
 - [**] Blaze: Support evergreen campaign creation. [https://github.com/woocommerce/woocommerce-ios/pull/13684]
 - [Internal] Custom fields: Renamed the OrderMetaData entity to MetaData [https://github.com/woocommerce/woocommerce-ios/pull/13736]
 - [Internal] Tooling: updated Sentry to avoid potential crash [https://github.com/woocommerce/woocommerce-ios/pull/13828]
+- [***] Login: Fix an issue with login using magic link [https://github.com/woocommerce/woocommerce-ios/pull/13926]
```

Comment:
Can you please move this under 20.2.1?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/13926/commits/6670ebdfe57eabea433564d152102f96743a73f1